velox/expression/CastExpr.cpp

```diff
@@ -47,6 +47,51 @@ const tz::TimeZone* getTimeZoneFromConfig(const core::QueryConfig& config) {
   return nullptr;
 }
 
+bool isIntegralType(const TypePtr& type) {
+  return type == TINYINT() || type == SMALLINT() || type == INTEGER() ||
+      type == BIGINT();
```

Comment:
It will be nice to have HUGEINT here, not for decimal but for just int128

Reply:
Is int128 used by any type other than the long decimal type in Presto? I assume it's only for the long decimal type.